lib/docs/index.html

```diff
@@ -284,6 +284,16 @@
         overflow-x: hidden;
       }
 
+      .fieldHasDocs {
+        margin-bottom: 0;
+      }
+
+      .fieldDocs {
+        border: 1px solid #2A2A2A;
+        border-top: 0px;
+        padding: 1px 1em;
```

Comment:
The `1px` horizontal padding here is necessary to avoid margin collapse when `border-top` is set to 0 (otherwise, the border wouldn't extend all the way up to the preceding `pre`).